### ISSUE TITLE
cut a release 

### ISSUE THREAD

**Issue by antholeole** [AshleyYakeley/NixVirt#68]
it seems the last release on flakehub was cut in April. pointing to master fixed my issue, but the README advises to not point to master (sensible!). Do you mind cutting a release? maybe 1.0, the api seems stable.

**Comment by AshleyYakeley** [comment]
Yeah I've been meaning to, but people keep submitting improvements... But I'll see what I can do.

**Comment by antholeole** [comment]
Thanks! But no worries it seems master is quite stable

**Comment by AshleyYakeley** [comment]
Done!
